packages/strapi/lib/core-api/__tests__/service.test.js

```diff
@@ -148,3 +167,23 @@ describe('Default Service', () => {
     });
   });
 });
+
+describe('getFetchParams', () => {
```

Comment:
@alexandrebodin I did some changes as requested, but as I was originally working on this task a few months ago, please read these test cases to make sure it works in the desired way 🙏